agent/engine/docker_image_manager_integ_test.go

```diff
@@ -81,23 +81,23 @@ func TestIntegImageCleanupHappyCase(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	imageState1 := imageManager.GetImageStateFromImageName(test1Image1Name)
-	if imageState1 == nil {
-		t.Fatalf("Could not find image state for %s", test1Image1Name)
-	} else {
+	imageState1, ok := imageManager.GetImageStateFromImageName(test1Image1Name)
+	if ok {
 		t.Logf("Found image state for %s", test1Image1Name)
-	}
-	imageState2 := imageManager.GetImageStateFromImageName(test1Image2Name)
-	if imageState2 == nil {
-		t.Fatalf("Could not find image state for %s", test1Image2Name)
 	} else {
-		t.Logf("Found image state for %s", test1Image2Name)
+		t.Fatalf("Could not find image state for %s", test1Image1Name)
```

Comment:
since you are here, can you change these to use `assert` or `require`?

Reply:
ok